core/server/master/src/test/java/alluxio/master/file/PermissionCheckTest.java

```diff
@@ -174,6 +176,7 @@ public FakeUserGroupsMapping() {
 
   @Before
   public void before() throws Exception {
+    Configuration.set(PropertyKey.MASTER_MOUNT_TABLE_ROOT_UFS, mTestFolder.newFolder());
```

Comment:
Is there anyway around this? Ideally we should not have configuration modification in `before`s